_layouts/couch.html

```diff
@@ -9,6 +9,7 @@
     {% endunless %}
 
     <header class="post-header">
+        <a href="{{page.email|to_gravatar}}"><img src="{{page.email|to_gravatar}}" alt="{{ name }}" class="post-avatar" /></a>
```

Comment:
You should add some big size to linked image, a link to 80px Gravatar is kinda useless. :-)